stable/kommander/templates/pre-deletion-purger.yaml

```diff
@@ -0,0 +1,91 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: purger-config
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-2"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+data:
+  purge.sh: |
+    #!/bin/bash
+    for ft in $(kubectl get federatedtypeconfigs -n kommander -o jsonpath='{range .items[*]}{.spec.federatedType.kind}.{.spec.federatedType.group}{"\n"}{end}') ; do
+      k delete ${ft} --all --all-namespaces
+    done
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: purger
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-2"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+
+rules:
+  - apiGroups:
+      - '*'
```

Comment:
Could we limit the scope here to the specific federated types API group?

Reply:
I explicitly made this a wildcard because if anyone adds a resource from an additional API group, this job would fail.